pkg/opensearch/lucene_handler.go

```diff
@@ -309,10 +310,12 @@ func getParametersFromServiceMapResult(smResult *client.SearchResponse) ([]strin
 			}
 		}
 	}
+
 	operations := make([]string, 0, len(operationMap))
 	for op := range operationMap {
 		operations = append(operations, op)
 	}
+	slices.Sort(operations)
```

Comment:
I feel like if sorting the operations benefitted users in someway it might make sense to do this, but if I understand correctly this is always going to be rendered as a graph where sorting won't really matter. I would vote we try sorting in the test if that's possible?